Composer/packages/server/src/controllers/project.ts

```diff
@@ -236,8 +242,15 @@ async function updateLgFile(req: Request, res: Response) {
 
   const currentProject = await BotProjectService.getProjectById(projectId, user);
   if (currentProject !== undefined) {
-    const lgFiles = await currentProject.updateLgFile(req.body.id, req.body.content);
-    res.status(200).json({ lgFiles });
+    const file = currentProject.lgFiles.find(d => d.id === req.body.id);
+    if (file && new Date(file.lastModified).getTime() > new Date(req.body.lastModified).getTime()) {
+      res.status(409).json({ message: 'This content was updated since it was loaded. Please reload.' });
+    } else {
+      const lastModified = await currentProject.updateLgFile(req.body.id, req.body.content);
+      res.status(200).json({ lastModified: lastModified });
+    }
+    // const lgFiles = await currentProject.updateLgFile(req.body.id, req.body.content);
```

Comment:
seems the behavior of this fucction has changed with this commented out

Reply:
It has changed. We no longer return the content of tall the LG files. As far as I can tell, this has no impact as the return value wasn't actually being used on the client side. I will review this again to make sure.